OsmAnd/res/values/strings.xml

```diff
@@ -9,6 +9,8 @@
          3. All your modified/created strings are in the top of the file (to make easier find what\'s translated).
     PLEASE: Have a look at http://code.google.com/p/osmand/wiki/UIConsistency, it may really improve your and our work  :-)  Thx - Hardy
     -->
+    <string name="show_closed_notes">Show closed notes</string>
+    <string name="switch_osm_notes_visibility_desc">Show/Hide OSM Notes on the map.</string>
```

Comment:
Am i wrong or is this string definition not matching the code?

Reply:
These are the strings for UI.